marbot.yml

```diff
@@ -259,6 +259,11 @@ Parameters:
     Type: String
     Default: true
     AllowedValues: [true, false]
+  ECSSpotNotifications:
+    Description: 'Receive a notification when an ECS Fargate Spot task gets interrupted or fails to launch.'
```

Comment:
Receive an alert, if an ECS Fargate Spot task gets interrupted or fails to launch.

Reply:
✅

Reply:
@andreaswittig ich glaube das hast du vergessen?

Reply:
✅